ui/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
I suggest we either write a more contained information here or remove this file and post required instructions in the main readme.